frontend/src/components/Landing/Landing.jsx

```diff
@@ -22,7 +22,7 @@ const Landing = () => {
       </div>
       <div className={styles.sectionLight}>
         <Section
-          title="Vous êtes psychologue ?"
+          title="Vous êtes psychologue ?"
```

Comment:
Unbreakable space added to avid having question mark alone in a second line on mobile